### DISCUSSION TITLE
Welcome to Tafe24S1T109_Ext Discussions!

### DISCUSSION THREAD

**Discussion by srothetafesa** [srothetafesa/Tafe24S1T109_Ext#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohnLSison** [comment]
Hey Samuel, are we going to be using TortoiseGit for this project, Gitbash or will be choosing whichever we feel comfortable with using when adding changes to the repository?

**Reply by srothetafesa** [reply]
Good question John, I was under the impression we were to use GitBash as that is what KT told us to use, however in order to push changes to the remote repository I believe we need to use TortiseGit. I have sent an email to KT to clarify.
